### ISSUE TITLE
Upgrade guide for EPS 0.2.0 (MacOs)

### ISSUE THREAD

**Issue by Pindol83** [HelgeHunding/guides#7]
Hi Drift,

can you upgrade the guide for install EPS for MacOS, i try to install it with your guide for 0.1.7, but when i create the automation script there is a problem (here an image of the error https://postimg.cc/TKxZdbXY).

thanks

Pindol

**Comment by DriftwoodPalace** [comment]
Try now!

**Comment by DriftwoodPalace** [comment]
Hi,

I just updated the guide with some new info, you might have to uninstall any old versions first. I am no Mac user, so a bit out on deep water here. But looks like you might have an old version of eps? `V 0.1.4`

That might interfere. Could you try using the command `sudo pip3 uninstall electrum-personal-server`

or `sudo pip3 uninstall electrum-personal-server 0.1.4`

And then try installing again. If that doesn't work. Maybe try navigating to the folder of the old version (if you have it).

`cd ~/electrum-personal-server-eps-v0.1.4` 

Or something and then

`sudo pip3 uninstall --user .`

And then installing the new version.

Please, report back if you find a solution. Then I can update the guide. If you don't find a solution, I have an old Mac lying around that I could try to reproduce the error on.

**Comment by DriftwoodPalace** [comment]
Read your issue again. So, yeah try to uninstall version 1.7.0 as well. I noticed I had the wrong files marked in the guide. Make sure you download these files as well https://driftwoodpalace.github.io/Hodl-Guide/images/64_eps-m.png

**Comment by Pindol83** [comment]
Hi,

i've uninstall electrum personal server 0.2.0 and then 0.1.4, and reinstall eps 0.2.0, but now i've another problem (image link https://postimg.cc/jCRhx2yx)

In the automated script i had wrote this `/Library/Python/3.7/bin/electrum-personal-server /Volumes/Pluto/es/electrum-personal-server-eps-v0.2.0/config.ini` (because the config file is ini and not cfg)

**Comment by DriftwoodPalace** [comment]
Hm, I don't really know here. But the two scripts are located in ~/Library/Python/3.7/bin I guess? And only those two and they were created today?

A somewhat related issue maybe https://github.com/chris-belcher/electrum-personal-server/issues/12

Make sure your Python version is clean. Maybe try to reinstall it.  That's all I could think of right now.

**Comment by Pindol83** [comment]
Yes correct the file are located in ~/Library/Python/3.7/bin (and created today). 

If i try to check where python3 is located with command "which python3" it report to me this folder /Library/Frameworks/Python.framework/Versions/3.7/bin/python3, with command "whereis python3" report to me this folder "/usr/bin/python3", could the problem be here?

**Comment by DriftwoodPalace** [comment]
Yes, that might be the case. Try this.

Open a terminal. Type `echo $PATH`

Make sure `/Library/Frameworks/Python.framework/Versions/3.7/bin` is at the top. If not you could try typing:

`export PATH=$PATH:"/Library/Frameworks/Python.framework/Versions/3.7/bin"`

Then it should use that Python first I think . If it still doesn't work maybe try to add `cd ~/Library/Python/3.7/bin` to a new line at the beginning of the script. Then it should definitely use the correct Python and we can rule that out if it doesn't work.

**Comment by Pindol83** [comment]
Finally i solve the problem, after unistall Python 3.7 and install Python 3.8, the problem still remain, with this error: ~/Library/Python/3.8/bin/electrum-personal-server", line 5, in <module>
    from electrumpersonalserver.server.common import main
ModuleNotFoundError: No module named 'electrumpersonalserver'

this file is located in ~/Library/Python/3.8/lib/python/site-packages/electrumpersonalserver/* 
but the folder was restricted to access for my user, i simply change the permissions of folder  ~/Library/Python/3.8/lib/python/site-packages/

Then the eps start correctly

**Comment by DriftwoodPalace** [comment]
Nice! I'll add that solution to the guide

**Comment by Pindol83** [comment]
Can I translate your guide in Italian and write an article on Medium? (obviously quoting your guide)

**Comment by DriftwoodPalace** [comment]
Absolutely, that would be awesome!

**Comment by Pindol83** [comment]
This is my translation if you want to link it on the bottom of your guide: https://medium.com/@pindol/installare-un-electrum-personal-server-su-macos-18d432eab37c

Do you plan to create a guide to install electrs in the future?

**Comment by DriftwoodPalace** [comment]
Cool!

I've added a link to the guide. 
No plans atm, I'm not really familiar with that project and feel like EPS is doing what I need for my "Hodl-guide".
